python/feather/ext.pyx

```diff
@@ -189,6 +189,18 @@ cdef class FeatherReader:
 
         return frombytes(cp.name()), values
 
+    def get_column_name(self, int i):
+        cdef:
+            unique_ptr[Column] col
+            Column* cp
+
+        if i < 0 or i >= self.num_columns:
+            raise IndexError(i)
+
+        check_status(self.reader.get().GetColumn(i, &col))
```

Comment:
This method potentially performs reads from the Feather file. This method will need to be added to the C++ library